_data/home.yml

```diff
@@ -192,3 +192,13 @@ projects:
   website: https://www.solo.io/products/service-mesh-hub/
   github: solo-io/service-mesh-hub
   description: Service Mesh Hub is a Kubernetes-native management plane that enables configuration and operational management of multiple heterogeneous service meshes across multiple clusters through a unified API.
+
+
+# Try to keep descriptions to about 50 words, for layout's sake
+training:
+- name: Envoy Fundamentals
+  logo: envoy-fundamentals-logo.svg
+  website: https://academy.tetrate.io/courses/envoy-fundamentals
+  description: Tetrate Academy introduces Envoy Fundamentals, a free course that allows all beginners to train in Envoy and experts to refresh their skills. The course offers a comprehensive tour of Envoy Proxy, complete with concept explanations, labs, and quizzes. If you pass all the quizzes, you also get a completion certificate to showcase at the end. 
+ 
+  
```

Comment:
hi - i meant a "newline" as opposed to blank line/s - we could probs do with some linting - apologies for hassle

Reply:
you can see that github is highlighting the issue

Reply:
Hey! Sorry, this is my first PR for an Open Source project.  Thank you for bearing with me! When you say 'newline' does it mean only one new line below the last written code? And not multiple lines as I did here?

Reply:
np re formatting issues, contributions are appreciated

essentially it means the last byte should be `\n` - most editors are configurable to do this auto on save for desired file types - this is probably what you want to look at